sdk/storage/azure-storage-blob-batch/pom.xml

```diff
@@ -93,7 +93,7 @@
     <dependency>
       <groupId>io.projectreactor</groupId>
       <artifactId>reactor-test</artifactId>
-      <version>3.3.9.RELEASE</version> <!-- {x-version-update;io.projectreactor:reactor-test;external_dependency} -->
+      <version>3.3.10.RELEASE</version> <!-- {x-version-update;io.projectreactor:reactor-test;external_dependency} -->
```

Comment:
even though this is just updating external_dependency, please get an acknowledgement from the storage team before merging this;

/cc @rickle-msft @gapra-msft